Mathlib/Algebra/Ring/Equiv.lean

```diff
@@ -864,24 +865,34 @@ theorem symm_trans_self (e : R ≃+* S) : e.symm.trans e = RingEquiv.refl S :=
   ext e.right_inv
 #align ring_equiv.symm_trans_self RingEquiv.symm_trans_self
 
+end RingEquiv
+
+namespace MulEquiv
+
 /-- If two rings are isomorphic, and the second doesn't have zero divisors,
 then so does the first. -/
```

Comment:
Did you keep the old docstring on purpose?

Reply:
Yeah, "two MulZeroClass" (or two monoids with zero?) would seem awkward and I expect these theorems to mostly be applied to rings. Also this file is about RingEquivs; even though I generalized to MulEquiv, isDomain and isField still requires Semiring, so they can't easily be moved.